modules/gcp_bigquery_table/outputs.tf

```diff
@@ -3,5 +3,5 @@ output "self_link" {
 }
 
 output "id" {
-  value = google_bigquery_table.google_bigquery_table.id
```

Comment:
Is this a breaking change?
We could instead add an additional output with `table_id`, right? so that current consumers won't break. Or is it currently not used by any consumer?

Reply:
currently not using, if someone is using it is actually breaks